test/usecases/global_breakpoint/leonhard.cpp

```diff
@@ -0,0 +1,163 @@
+/*********************************************************
```

Comment:
I think that before merging this PR this file should be deleted since it's not contributing to any test. If #1131 gets merged first with proper tests on those "golden" reference files then it can be added

Reply:
Probably, yes. However, did it help with reviewing this PR?